package.json

```diff
@@ -19,7 +19,7 @@
     "test:yorkie.dev": "karma start karma.conf.js --single-run --testRPCAddr=https://yorkie.dev/api",
     "test:watch": "karma start karma.conf.js",
     "lint": "eslint . --fix --max-warnings=0 --ext .ts",
-    "prepare": "npm run build"
+    "prepare": "ts-patch install && npm run build"
```

Comment:
it seems to not showing generated docs file including missing one 🤔 I followed the steps to take a look our reference document locally
1. npm run build
2. npm run predoc
3. npm run build:docs

And I got
<img width="288" alt="Screen Shot 2021-08-25 at 9 22 15 AM" src="https://user-images.githubusercontent.com/4694139/130706633-fe18ca51-9a3f-4901-ae7e-a082947960e2.png">
But it should be the follwoing image
<img width="291" alt="Screen Shot 2021-08-25 at 9 22 45 AM" src="https://user-images.githubusercontent.com/4694139/130706673-079baa36-2b76-42c7-b19c-a738b1d95bb5.png">

Reply:
That's weird. It works well even when tested in a fresh environment.

![image](https://user-images.githubusercontent.com/2173789/130712532-6e095f97-d2e5-4b54-9f0e-e25e256bdeae.png)


Could you try `ts-patch check`? My result is like this:
```
Checking TypeScript v3.9.10 installation in .../yorkie-js-sdk/node_modules/typescript

[+] tsc.js is patched with ts-patch version 1.4.3 
[-] tsserverlibrary.js is not patched.
[+] typescript.js is patched with ts-patch version 1.4.3 
[-] typescriptServices.js is not patched.
[-] tsserver.js is not patched.
```

Reply:
oh my bad. I tested it in different branch 😅 thanks!